app/scripts/controllers/rooms_controller.js

```diff
@@ -0,0 +1,34 @@
+(function(){
+  'use strict';
+
+  angular
+    .module('app.controllers')
+    .controller('RoomsController', RoomsController);
+
+  RoomsController.$inject = ['$scope', '$location', 'DataFactory'];
+
+  function RoomsController($scope, $location, DataFactory){
+    $scope.room = {};
```

Comment:
To take advantage of `controllerAs` syntax, I think you don't have to use `$scope$ for data holding